File: Week1/mario_less.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Ask user to input height
    int height;
    do
    {
        printf("Please input a positive integer between 1 and 8. \n");
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);


    //Print the pyramid from i = 1 to height
    //Make sure there are spaceholders j = height - i
    for (int i = 1; i <= height; i++)
    {
        for (int j = 1; j <= (height - i); j++)
        {
            printf(" ");
        }

        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week6/mario_less.py
# TODO

from cs50 import get_int


def main():
    # ask user to input height
    height = get_int('Height: ')

    if height < 1 or height > 8:
        print('Please input a positive integer between 1 and 8, inclusive.')
        # restart the function, so that if user input a valid height, the program will continue to print
        main()
    else:
        # print the pyramid from i = 1 to height
        for i in range(1, height + 1):
            # there should be j spaces where j = height - i
            for j in range(height - i):
                print(" ", end="")
            for k in range(i):
                print("#", end="")
            print()


main()